### ISSUE TITLE
Rails/InverseOf cop false positive for polymorphic assocations

### ISSUE THREAD

**Issue by wintersolutions** [bbatsov/rubocop#5329]
Rubocop 0.52.0 complains about missing 'inverse_of' for polymorphic `belongs_to` association. According to the rails documentation `inverse_of` is not a valid option for `polymorphic: true` assocations.

https://github.com/bbatsov/rubocop/issues/5236 seems to adress this issue in the [comments](https://github.com/bbatsov/rubocop/issues/5236#issuecomment-351737179). But after a quick glance at the merged source code it only seems to address the `with_options` false positive.

--------

## Expected behavior

Ignore missing invalid option.

## Actual behavior

`Rails/InverseOf` suggests to add the invalid option

## Steps to reproduce the problem

Let rubocop check the following example file:

```
module Example
  class Foo < ApplicationRecord
    belongs_to :placeable, polymorphic: true
  end
end
```
Its getting reported.

## RuboCop version

```
$ rubocop -v
0.52.0
```

**Comment by wata727** [comment]
Hi @wintersolutions, I fixed this false positive in https://github.com/bbatsov/rubocop/pull/5297.
Please try v0.52.1. This problem has been fixed :)

## 0.52.0

```
$ rubocop _0.52.0_ --rails --only Rails/InverseOf app/models/foo.rb
Inspecting 1 file
C

Offenses:

app/models/foo.rb:3:5: C: Rails/InverseOf: Specify an :inverse_of option.
    belongs_to :placeable, polymorphic: true
    ^^^^^^^^^^

1 file inspected, 1 offense detected
```

## 0.52.1

```
$ rubocop _0.52.1_ --rails --only Rails/InverseOf app/models/foo.rb                                                                                         [22:52:28]
Inspecting 1 file
.

1 file inspected, no offenses detected
```

**Comment by wintersolutions** [comment]
Ah I looked at the wrong issue/commit. Its working. Thanks a lot!
